### DISCUSSION TITLE
Laravel Documentation

### DISCUSSION THREAD

**Discussion by emreakay** [laravel/framework#38634] [Q&A]
Hi,

I want to create a documentation like laravel's docs.
How can i create a static website from .md files inside a git repo?

Thanks.

**Comment by dawidholka** [comment]
@emreakay 
Here is the [source code of the laravel.com](https://github.com/laravel/laravel.com-next) website you can check how this has been implemented.

**Comment by adityadees** [comment]
> Hi,
> 
> I want to create a documentation like laravel's docs.
> How can i create a static website from .md files inside a git repo?
> 
> Thanks.

I think you ask about **How to host static markdown?**
There are many ways to do it. But if you want to host on GitHub the simplest you can use the settings->github page. You can choose a template from github for easy use.

**Comment by reindert-vetter** [comment]
For https://confetti-framework.com/ I used Vuepress (hosting on GitHub pages). I'm using Algolia to let a user search through the documentation. I have registered a domain and point it to Github Pages.

I just need to run `deploy.sh` to deploy an update. Feel free to use it for inspiration: https://github.com/confetti-framework/docs

(Install Algolia when you have already linked a domain to it)
